README.md

```diff
@@ -1,30 +1,33 @@
-<img width="500px" src="https://github.com/user-attachments/assets/1e16f1fd-516a-4538-9699-cd96ce89113c">
+<img width="500px" src="https://github.com/user-attachments/assets/ba3d33f5-58a1-4d17-b097-7bf6b589d7e1">
 
-# AtB - Application Mobile
+# AtB app
 
-This repo contains documentation and code for the AtB app with travel assistance and ticketing.
+This repo contains code for the AtB app, a travel assistant and ticketing app for public transportation and mobility in Norway. In addition to AtB in Trøndelag, the app is also used by [FRAM](https://frammr.no/) (Møre og Romsdal), [Reis Nordland](https://www.reisnordland.no/) and [Svipper](https://svipper.no/) (Troms).
 
 You can submit a new bug report or feature proposal by [creating a new issue](https://github.com/AtB-AS/mittatb-app/issues/new/choose).
 
 ## Contribution
 
-We love feedback and suggestions. The AtB app and service is continously improved over time by working with users and observing real usage. If you have any issues or suggestions for improvement we would also love Pull Requests. See our [contribution guide](./CONTRIBUTING.md).
+We love feedback and suggestions. The AtB app is continously improved over time by working with users and observing real usage. If you have any issues or suggestions for improvement we would also love Pull Requests. See our [contribution guide](./CONTRIBUTING.md).
 
 ## Getting started
 
+> [!NOTE]
+> For external contributors, we should set up default environment files to make it easier to run the app without access to git-crypt secrets and Firebase.
```

Comment:
Note - the git-crypt file is planned to be removed completely: https://github.com/AtB-AS/kundevendt/issues/19057